.github/workflows/deploy-release.yml

```diff
@@ -0,0 +1,54 @@
+name: Update shared-sites dependent website repositories on release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  update-repositories:
+    name: Update dependent website repositories on release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Git
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Extract release version
+        id: extract_version
+        run: echo "BOOTSTRAP_VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+
+      - name: Clone target repositories
+        run: |
+          repositories=(
+            "keymanapp/api.keyman.com"
+            "keymanapp/help.keyman.com"
+            "keymanapp/keyman.com"
+            "keymanapp/keymanweb.com"
+            "keymanapp/s.keyman.com"
+          )
+          for repo in "${repositories[@]}"; do
+            git clone "https://github.com/$repo.git"
+          done
+
+      - name: Update build.sh files
+        run: |
+          for repo in keymanapp/*; do
```

Comment:
Do we really have a `keymanapp` subdirectory?

Reply:
No, we do not. Am fixing. :grin: